src/netex_idf/offre.rs

```diff
@@ -0,0 +1,82 @@
+use crate::{model::Collections, Result};
+use failure::format_err;
+use log::{info, warn};
+use minidom::Element;
+use std::{fs::File, io::Read, path::Path};
+use walkdir::WalkDir;
+
+pub fn read_offre_folder<P: AsRef<Path>>(
```

Comment:
if this function is only called by `read` function, is it necessary to have `P: AsRef<Path>` ?

Reply:
Yes, it's still a `pub` function but at the same time, it's not really exposed on `transit_model` (only internally). I removed it for the sake of simplicity of the code in 292355f35b9e1b51430d00f88e73e1be15b69ce9.